dspace/config/clarin-dspace.cfg

```diff
@@ -281,3 +281,5 @@ download.all.limit.min.file.count = 1
 download.all.limit.max.file.size = 1073741824
 # minimum total size of files for enabling download alert:
 download.all.alert.min.file.size = 10485760
+# used in elg crosswalk exposing download locations
+elg.downloadLocation.exposed = 0
```

Comment:
I think it would be better to use kebab case `download-location` instead of camel case `downloadLocation` to have the cfg properties names consistent.

Reply:
changed